starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,65 @@
 // Names and Input
-
+const hacker1 = 'Marc';
+console.log('The driver\'s name is ' + hacker1);
+const hacker2 = 'Llorenç';
+console.log('The navigator\'s name is ' + hacker2);
+const hacker3 = 'Francesc';
+console.log('The navigator\'s name is ' + hacker3);
 
 //Conditionals
+if ( hacker1.length == hacker2.length){
+  console.log('Wow! you both got equally long names, ' + hacker1.length + ' characters');
+} else if (hacker1.length > hacker2.length){
+  console.log('The Driver has the longest name, it has ' + hacker1.length + ' characters');
+} else{
+  console.log('Yo, navigator got the longest name, ' + hacker2.length + ' characters');
+}
+
+
+//Loops
+var nameCapital='';
+for(let i=0; i < hacker3.length; i++){
```

Comment:
si utilizamos ES6 con let en los for siempre es mejor utilizar const para el resto de variables si no se van a modificar. En caso de que se modifiquen utilizar let. 
Como regla general: Siempre ```const``` hasta que el código de error.